readme.md

```diff
@@ -56,9 +63,15 @@ There are three main reasons for this:
 
 ---
 
-### Potential Project Ideas
+### Project Ideas
+
+For this project, work with your team to build a creative product that you actually think someone will want to use.
+
+All students are required to submit a project 3 idea as an issue [here](https://github.com/ga-wdi-exercises/project3/issues). The deadline for submission is March 10 at 9:00AM.
+
+After groups have been announced during project 3 kickoff, teams will have one hour to select an idea from all submissions. Students will then inform their assigned instructor of the idea their team has chosen. Ideas will be approved on a first-come, first-serve basis.
 
-For this project, work with your team to build a creative product that you actually think someone will want to use. We won't have time to do tons of customer research, but take some time to brainstorm. If you're struggling for ideas, you can check out the [GA Project Gallery](http://gallery.ga.co).
+**Teams may not choose an idea that any group member has submitted.**
```

Comment:
"any group member" --> "one of their team members"

The current way makes it sound like you can't pick an idea that any group members, regardless of which group they belong to, has submitted.